conn.go

```diff
@@ -40,22 +53,33 @@ func (c *Conn) WriteUnreliablePacket(buf []byte) error {
 }
 
 func (c *Conn) Read(header PacketHeader, buf []byte) error {
-	buf = c.protocol.ReadPacket(header, buf)
-
-	if len(buf) != 0 {
```

Comment:
warning: exported method Conn.WriteUnreliablePacket should have comment or be unexported (golint)